src/components/MultiAssetPreview/MultiAssetPreview.spec.tsx

```diff
@@ -0,0 +1,93 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import { mount } from "@cypress/react";
+import React from "react";
+import { ImageAsset } from "../../types/assets";
+import { MultiAssetPreview } from "./MultiAssetPreview";
+
+const MULTI_ASSET_PREVIEW_ID = "[data-test-id=multi-asset-preview]";
+const ASSETS_AMOUNT_ID = "[data-test-id=assets-amount]";
+const NUMBER_OF_LOCATIONS_ID = "[data-test-id=number-of-locations]";
+const ASSETS_IMAGE_ID = "[data-test-id=assets-image]";
+
+const EXAMPLE_IMAGES: ImageAsset[] = [
+    {
+        name: "foo1",
+        size: 2000,
+        type: "image",
+        extension: "JPG",
+        src: "https://picsum.photos/100/150",
+    },
+    {
+        name: "foo2",
+        size: 2000,
+        type: "image",
+        extension: "JPG",
+        src: "https://picsum.photos/200/200",
+    },
+    {
+        name: "foo3",
+        size: 2000,
+        type: "image",
+        extension: "JPG",
+        src: "https://picsum.photos/100/100",
+    },
+    {
+        name: "foo4",
+        size: 1000,
+        type: "image",
+        extension: "JPG",
+        src: "https://picsum.photos/400/200",
+    },
+];
+
+describe("MultiAssetPreview Component", () => {
+    it("should render without images", () => {
+        mount(<MultiAssetPreview onClick={() => alert("Multi Asset clicked")} />);
+
+        cy.get(MULTI_ASSET_PREVIEW_ID).should("exist");
+    });
+
+    it("should render with images", () => {
+        mount(
+            <MultiAssetPreview
+                onClick={() => alert("Multi Asset clicked")}
+                numberOfLocations={12}
+                assetsAmount={25}
+                images={EXAMPLE_IMAGES}
+            />,
+        );
+
+        cy.get(MULTI_ASSET_PREVIEW_ID);
+    });
+
+    it("image urls should exist", () => {
+        mount(<MultiAssetPreview onClick={() => alert("Multi Asset clicked")} images={EXAMPLE_IMAGES} />);
+
+        cy.get(ASSETS_IMAGE_ID).should("have.attr", "style").and("include", "background-image: url(");
```

Comment:
I'd move this to the `should render with images` test and check if the urls are correct (you can do it in the same statement), e.g.
```

cy.get(ASSETS_IMAGE_ID).should("have.attr", "style", ""background-image: url('https://picsum.photos/100/150')")
```